web/src/beta/features/ProjectSettings/innerPages/PublicSettings/index.tsx

```diff
@@ -52,6 +52,7 @@ export type SettingsProject = {
   isArchived: boolean;
   enableGa: boolean;
   trackingId: string;
+  __typename?: string;
```

Comment:
```suggestion
  typeName?: "project" | "story";
```
We could make it more clear i guess.

Reply:
fix